bin/plugin/lib/version.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+// @ts-ignore
+const semver = require( 'semver' );
```

Comment:
Could optionally install [`@types/semver`](https://www.npmjs.com/package/@types/semver) 🤷 (Give you type-checking and autocompletion on `semver.parse`)